tutoraspects/templates/aspects/build/aspects-superset/openedx-assets/assets/datasets/query_log.yaml

```diff
@@ -118,7 +118,7 @@ sql: |
       query,
       http_user_agent
   FROM system.query_log
-  WHERE (has(databases, '{{ASPECTS_XAPI_DATABASE}}') OR has(databases, '{{ASPECTS_EVENT_SINK_DATABASE}}') OR has(databases, '{{DBT_PROFILE_TARGET_DATABASE}}')) AND (http_user_agent LIKE 'aspects-%') AND (type = 'QueryFinish')
+  WHERE databases <> ['system'] AND (http_user_agent LIKE 'aspects-%') AND (type = 'QueryFinish')
```

Comment:
I'm not sure how much it matters, but there are some installs that have other things in ClickHouse aside from Aspects and this change might make things slower there.